lib/puppet-lint/plugins/check_whitespace/colon_whitespace_before.rb

```diff
@@ -0,0 +1,22 @@
+# Public: Check the manifest tokens for resource type declaration that has no whitespace before a colon and record a warning for each instance found.
+# #
+# https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace
```

Comment:
That URL redirects to https://docs.puppet.com/puppet/latest/style_guide.html#spacing-indentation-and-whitespace, it's probably better to point directly at it.